plugin/src/main/java/net/dzikoysk/funnyguilds/guild/Guild.java

```diff
@@ -164,16 +161,28 @@ public void setRegion(Region region) {
     }
 
     public boolean isSomeoneInRegion() {
```

Comment:
To raczej powinien być kod w regionie/serwisie regionów

Reply:
Ta... ale zastanawia mnie czy ta metoda powinna tak wyglądać? Bo wymaga ona:
A) konfiguracji regionów i regionu -> Region/RegionManager
B) listy członków Gildii -> Guild/GuildManager

jedna z tych rzeczy powinna zostać wyłączona i wydaje mi się że to może być `B`?